restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/UssdPullTest.java

```diff
@@ -438,6 +440,40 @@ public void testUssdMessageLengthExceeds() {
         bobCall.dispose();
     }
 
+    @Test
+    public void testUssdPullClosedAccount() {
+        final SipCall bobCall = bobPhone.createSipCall();
+        bobCall.initiateOutgoingCall(bobContact, ussdPullDidClosedAccount, null, UssdPullTestMessages.ussdClientRequestBody, "application", "vnd.3gpp.ussd+xml", null, null);
+        assertLastOperationSuccess(bobCall);
+
+        assertTrue(bobCall.waitOutgoingCallResponse(5 * 1000));
+        int responseBob = bobCall.getLastReceivedResponse().getStatusCode();
+        if (responseBob == Response.TRYING) {
+            assertTrue(bobCall.waitOutgoingCallResponse(5 * 1000));
+            assertTrue(bobCall.getLastReceivedResponse().getStatusCode() == Response.NOT_ACCEPTABLE);
```

Comment:
lets try to use assertEquals instead of this. Is pretty bad when test is suddenly broken, and fail message doesnt provide a hint.

Reply:
Agree